docs/admin_api/rooms.md

```diff
@@ -76,7 +76,7 @@ GET /_synapse/admin/v1/rooms
 
 Response:
 
-```
+```jsonc
```

Comment:
What's `jsonc` vs. `json`? (It seems to work, just not seen this before.)

Reply:
I think its json with comments?

Reply:
It is necessary for lines like `... (8 hidden items) ...`
Otherwise they will be marked as errors.

Reply:
Great trick! Thanks!